utils/network.go

```diff
@@ -29,13 +29,13 @@ func (n Network) String() string {
 func (n Network) URL() string {
 	switch n {
 	case GOERLI:
-		return "https://alpha4.starknet.io"
+		return "https://alpha4.starknet.io/feeder_gateway/"
 	case MAINNET:
-		return "https://alpha-mainnet.starknet.io"
+		return "https://alpha-mainnet.starknet.io/feeder_gateway/"
 	case GOERLI2:
-		return "https://alpha4-2.starknet.io"
+		return "https://alpha4-2.starknet.io/feeder_gateway/"
 	case INTEGRATION:
-		return "https://external.integration.starknet.io"
+		return "https://external.integration.starknet.io/feeder_gateway/"
```

Comment:
I considered doing this as well, but I was not sure if there were other endpoints open to public

Reply:
Hmmm, that's I good point that I did not consider. I am not aware of any. If that is the case, we could make another function `GatewayUrl` that returns `network.Url() + "/feeder_gateway/"`.